lib/content-services/document-list/components/content-action/content-action.component.ts

```diff
@@ -65,7 +65,7 @@ export class ContentActionComponent implements OnInit, OnChanges {
 
     /** Is the menu item disabled? */
```

Comment:
This JSDoc and the one for the `target` property should be the same as the property table descriptions in the Markdown file. Easiest way to get them in sync is to add the JSDoc and do "npm run docbuild -- -p enhance /docs/content-services/content-action.component.md". This will copy all the JSDocs to the .md file.

Reply:
I restored original description for now, thanks.